lib/utils.sh

```diff
@@ -60,3 +60,8 @@ get_download_file_path() {
 
     echo "$tmp_download_dir/$pkg_name"
 }
+
+get_latest_luarocks_version() {
+    curl -sL "https://api.github.com/repos/luarocks/luarocks/tags?per_page=1&page=1" |
+        grep '"name"' | cut -d\" -f4 | cut -c2-
```

Comment:
Looks good to me. The only downside to this approach is that it appears to rely on the whitespace being present in the response, which is fine, but something to keep in mind.

Reply:
Yeah, I would have preferred to use `jq` to pick out the name, but I wasn't sure if it was worth adding another dependency. I'm happy to add it and put a note in the README, if you think it's not a big deal.